doc/src/manual/multi-threading.md

```diff
@@ -388,6 +388,14 @@ threads in Julia:
     This may require some transitional work across the ecosystem before threading
     can be widely adopted with confidence. See the next section for further details.
 
+## [Task Migration](@id man-task-migration)
```

Comment:
Maybe add a compat note or something that it was introduced in v1.7?

Reply:
yeah. Good point, done